functions/notify_slack.py

```diff
@@ -15,6 +15,15 @@ def decrypt(encrypted_url):
   except Exception:
     logging.exception("Failed to decrypt URL with KMS")
 
+def get_ssm_param(ssm_param):
+  try:
+    ssm = boto3.client("ssm")
+    return ssm.get_parameter(
+      Name=ssm_param,
+      WithDecryption=True
+    ).get("Parameter").get("Value")
+  except Exception:
+    logging.exception("Failed to get URL from SSM")
```

Comment:
Probably need to raise an exception here and possible change the message to `Failed to get Slack webhook URL from SSM`